astroquery/astorbdb/core.py

```diff
@@ -0,0 +1,720 @@
+                                          timeout=self.TIMEOUT, cache=cache)
+
+        response['albedos'] = self._request('GET',
+                                            url=self.URL + object_name + '/data/albedos',
+                                            timeout=self.TIMEOUT, cache=cache)
+
+        response['colors'] = self._request('GET',
+                                           url=self.URL + object_name + '/data/colors',
+                                           timeout=self.TIMEOUT, cache=cache)
+
+        response['taxonomies'] = self._request('GET',
+                                               url=self.URL + object_name + '/data/taxonomies',
+                                               timeout=self.TIMEOUT, cache=cache)
+
+        response['lightcurves'] = self._request('GET',
+                                                url=self.URL + object_name + '/data/lightcurves',
+                                                timeout=self.TIMEOUT, cache=cache)
+
+        response['dynamicalfamily'] = self._request('GET',
+                                                    url=self.URL + object_name + '/data/dynamical-family',
+                                                    timeout=self.TIMEOUT, cache=cache)
+
+        response['escaperoutes'] = self._request('GET',
+                                                 url=self.URL + object_name + '/data/escape-routes',
+                                                 timeout=self.TIMEOUT, cache=cache)
+
+        if get_raw_response:
+            self._return_raw = True
+
+        if get_uri:
+            self._uri = {'designations': response['designations'].url,
+                         'elements': response['elements'].url,
+                         'orbit': response['orbit'].url,
+                         'albedos': response['albedos'].url,
+                         'colors': response['colors'].url,
+                         'taxonomies': response['taxonomies'].url,
+                         'lightcurves': response['lightcurves'].url,
+                         'dynamicalfamily': response['dynamicalfamily'].url,
+                         'escaperoutes': response['escaperoutes'].url
+                         }
+
+        return response
+
+    def _parse_result(self, response, *, verbose=None):
+        """
+        Parser for astorbDB AstInfo request results
+        """
+
+        if self._return_raw:
+            return response.text
+
+        # decode json response from Lowell astorbDB into ascii
+        try:
+            if self.query_type == 'all_astinfo':
+                src = OrderedDict()
+                for key in response:
+                    src[key] = OrderedDict(json.loads(response[key].text))
+            else:
+                src = OrderedDict(json.loads(response.text))
+        except ValueError:
+            raise ValueError('Server response not readable.')
+
+        if self.query_type == 'designations':
```

Comment:
`if ... elif ... elif` would be better: indicates that only one match can occur, and allows for a final `else` that can raise a `ValueError` in case of no match.